src/analyzer/analyzerscheduledtrack.cpp

```diff
@@ -0,0 +1,16 @@
+#include "analyzer/analyzerscheduledtrack.h"
+
+#include "analyzer/analyzertrack.h"
+#include "track/trackid.h"
+
+AnalyzerScheduledTrack::AnalyzerScheduledTrack(TrackId trackId, AnalyzerTrack::Options options)
+        : trackId(trackId), options(options) {
+}
+
+const TrackId AnalyzerScheduledTrack::getTrackId() const {
+    return trackId;
+}
+
+const AnalyzerTrack::Options AnalyzerScheduledTrack::getOptions() const {
```

Comment:
I have no strong opinion on whether by-ref or by-value is better here, it's not a large type currently (2 bytes), but since it might get bigger in the future returning by ref might make sense to future-proof this against large copies (though it's probably not accessed that often anyways).